src/Server_TEST.cc

```diff
@@ -759,6 +759,25 @@ TEST_P(ServerFixture, GetResourcePaths)
   EXPECT_EQ("/home/user/another_path", res.data(1));
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, CachedFuelWorld)
+{
+  auto logPath = common::joinPaths(
+      std::string(PROJECT_BINARY_PATH), "test_log_path");
+  auto cachedWorldPath =
+    common::joinPaths(std::string(PROJECT_SOURCE_PATH), "/test/worlds");
```

Comment:
```suggestion
    common::joinPaths(std::string(PROJECT_SOURCE_PATH), "test", "worlds");
```

I think Windows will like this better, even though this test probably doesn't pass on Windows anyway.

The same goes for the one below. `common::joinPaths` will use the `\` separator on Windows, so it becomes `fuel.ignitionrobotics.org\OpenRobotics\worlds\Test%20world\2\test.sdf`

Reply:
Ahh oops, I made an incorrect assumption that `joinPaths` would reformat slashes to be compatible with the system, updated in b476572905872ed0a9ed35f9595643345d26691e